src/custom/pages/Swap/ArrowWrapperLoader.tsx

```diff
@@ -0,0 +1,136 @@
+import React from 'react'
+import styled, { css } from 'styled-components'
+import loadingCowGif from 'assets/cow-swap/cow-load.gif'
+import useLoadingWithTimeout from 'hooks/useLoadingWithTimeout'
+import { useIsQuoteRefreshing } from 'state/price/hooks'
+
+export interface ArrowWrapperProps {
+  children: React.ReactNode
+}
+
+export function ArrowWrapperLoader({ children }: ArrowWrapperProps) {
+  const COW_LOADING_TIME = 4000
```

Comment:
make a const in `custom/constants/index` not here. also don't push constants like this inside the component, move it outside